app/onepassword_events_api/appserver/static/javascript/components/wizard.js

```diff
@@ -0,0 +1,179 @@
+import React from "react";
+
+const VERSION = "1.6.0";
+
+const e = React.createElement;
+export default class Wizard extends React.Component {
```

Comment:
Does the version of react support hooks? If yes, we should probably use functional components where we can over class components. If not, disregard :)